prime-router/docs/api/README.md

```diff
@@ -1,16 +1,79 @@
-This folder holds open api docs resources:
+# Swagger UI
+
+Swagger UI is a tool that allows you to visually interact with ReportStream APIs that have been defined using the OpenAPI
+Specification (OAS). It provides a user-friendly interface that makes it easier to understand and explore capabilities of
+the API without requiring external tools such as curl or Postman.
+
+This directory contains all open api spec resources:
 
 1. ./: specs in yaml format derived from other process (vs auto generated specs under ./generated)
```

Comment:
probably remove any mention of these manually curated yml files since they do not show up in swagger and will be replaced with annotations.